packages/smooth_app/lib/pages/product/nutrition_page_loaded.dart

```diff
@@ -228,15 +228,15 @@ class _NutritionPageLoadedState extends State<NutritionPageLoaded> {
         crossAxisAlignment: CrossAxisAlignment.center,
         children: <Widget>[
           SizedBox(
-            width: getColumnSizeFromContext(context, 0.6),
+            width: getColumnSizeFromContext(context, 0.55),
```

Comment:
A clean solution would use `Expanded` and `flex`:
* we set a fixed width for the unit column (e.g. `getColumnSizeFromContext(context, 0.3)`
* and we let flutter compute the correct size with `Expanded` and `flex`

Reply:
Yes! This is a better solution. Thankyou @monsieurtanuki .